lib/mrsk/cli/traefik.rb

```diff
@@ -1,24 +1,25 @@
 class Mrsk::Cli::Traefik < Mrsk::Cli::Base
   desc "boot", "Boot Traefik on servers"
-  def boot(login: true)
+  def boot
     mutating do
       on(MRSK.traefik_hosts) do
-        execute *MRSK.registry.login if login
+        execute *MRSK.registry.login
         execute *MRSK.traefik.run, raise_on_non_zero_exit: false
       end
     end
   end
 
+  method_option :rolling, type: :boolean, default: false
```

Comment:
We use `option` which is just an alias for `method_option` everywhere else - could you update for consistency?